packages/salesforcedx-visualforce-language-server/test/unit/javascriptMode.test.ts

```diff
@@ -46,7 +46,8 @@ describe('HTML Javascript Support', () => {
     }
   }
 
-  it('Should display completions', () => {
+  it('Should display completions', function() {
+    this.timeout(30000);
```

Comment:
This was regularly timing out. I didn't dig at all on figuring out what it's waiting on locally but it now passes without a significant change in the run time.